src/proxy-page/steps/addJira.ts

```diff
@@ -211,135 +238,178 @@ export default (config: ConfigService, jiraService: JiraService): Step => async
       // Load new base URL if defined a specific connection for Jira as ENV variables
       // otherwise default to standard baseURL defined for main server
       const baseUrl = process.env[`CPV_JIRA_${server.replace(/\s/, '_')}_BASE_URL`]
-          ?? config.get('confluence.baseURL');
+        ?? config.get('confluence.baseURL');
       issues.forEach((issue) => {
         data.push({
-          key: {
-            name: issue.key,
-            link: `${baseUrl}/browse/${issue.key}?src=confmacro`,
-          },
+          reporter: issue.fields.reporter?.displayName || '',
+          components: getContentWithMap(issue.fields.components),
+          acceptance_criteria: getTextContent(issue.fields.customfield_10042),
+          detail_design_reference: getTextContent(issue.fields.customfield_10129),
+          storypoints: issue.fields.customfield_10026 || '',
+          labels: issue.fields.labels || [],
+          sprint: getContentWithMap(issue.fields.customfield_10020),
```

Comment:
This `customfield_10020` looks like a custom field for us.
What would happen if we run this code in another Jira/Confluence instance?